pheme/views.py

```diff
@@ -1,22 +1,37 @@
-from dataclasses import dataclass
-from rest_framework import generics, serializers
-
-
-@dataclass
-class Test:
-    name: str
-
-
-class TestSerializer(serializers.Serializer):
-    name = serializers.CharField()
-
-    def update(self, instance, validated_data):
-        raise NotImplementedError('`update()` must be implemented.')
-
-    def create(self, validated_data):
-        raise NotImplementedError('`create()` must be implemented.')
-
-
-class TestView(generics.ListAPIView):  # new
-    queryset = iter([Test("1"), Test("2")])
-    serializer_class = TestSerializer
+import dataclasses
+
+from rest_framework.decorators import api_view, parser_classes
+from rest_framework.response import Response
+from pheme.parser.xml import XMLParser
+from pheme.transformation import scanreport
+
+
+@api_view(['POST'])
+@parser_classes([XMLParser])
+def report(request):
+    return Response(request.data)
+
+
+@api_view(['POST'])
+@parser_classes([XMLParser])
+def template(request):
+    # Threat: None, Low, Medium, High
+    input_flavour = request.GET.get('flavour', 'gvmd')
+    grouping = request.GET.get('grouping')
+    print(
+        'selecting input flavour: {} and grouping {}'.format(
+            input_flavour, grouping
+        )
+    )
```

Comment:
Should that stay or is it just a debug statement?

Reply:
thanks, deleted